content/labpubs.bib

```diff
@@ -1,4 +1,62 @@
+  journal={arXiv preprint arXiv:1707.04393},
+  year={2017}
+}
+@inproceedings{hagenah2017combining,
+  title={Combining Deformation Modeling and Machine Learning for Personalized Prosthesis Size Prediction in Valve-Sparing Aortic Root Reconstruction},
+  author={Hagenah, Jannis and Scharfschwerdt, Michael and Schweikard, Achim and Metzner, Christoph},
+  booktitle={International Conference on Functional Imaging and Modeling of the Heart},
+  pages={461--470},
+  year={2017},
+  organization={Springer}
+}
+@inproceedings{maeki2017a,
+  title= {Layer V pyramidal cells as mediators of delta oscillations: Insights from biophysically detailed modeling and connections with schizophrenia genetics},
+  author={M\"aki-Marttunen, T and Krull, F and Bettella, F and Metzner, C and Devor, A and Djurovic, S and Dale, AM and Andreassen, OA and Einevoll, GT},
+  booktitle={BMC Neuroscience, 18(Suppl.1)},
+  year={2017},
+}
+@inproceedings{metzner2017a,
+  title= {A Framework for Automated Validation and Comparison of Models of Neurophysiological and Neurocognitive Biomarkers of Psychiatric Disorders. },
+  author={Metzner, C and Schweikard, A and M\"aki-Marttunen, T and Zurowski, B and Steuber, V},
+  booktitle={BMC Neuroscience, 18(Suppl.1)},
+  year={2017},
+}
+@inproceedings{goncharenko2017,
+  title= {Modulation of epileptic activity in thalamo-cortical networks by input from the cerebellar nuclei. },
+  author={Goncharenko, J and Kros, L and Davey, N and Metzner, C and De Zeeuw, C and Steuber, V},
+  booktitle={BMC Neuroscience, 18(Suppl.1)},
+  year={2017},
+}
+@inproceedings{sinha2017,
+  title= {The effect of homeostatic structural plasticity on associative memory in a network with spike-time dependent inhibitory synaptic plasticity.},
+  author={Sinha, A and Metzner, C and Adams, R and Schmuker, M and Davey, N and Steuber, V},
+  booktitle={BMC Neuroscience, 18(Suppl.1)},
+  year={2017},
+}
+@article{metzner2017b,
+title={[Re] Modeling GABA alterations in schizophrenia: a link between impaired inhibition and altered gamma and beta range auditory entrainment.}
+author={Metzner, Christoph}
+journal={ReScience}
+year={2017},
+volume={3},
+number={1},
+}
+@inproceedings{maeki2017b,
+  title= {Delta oscillations in schizophrenia: Insights from biophysically detailed modeling of networks of layer V pyramidal cells.},
+  author={M\"aki-Marttunen, T and Krull, F and Bettella, F and Metzner, C and Devor, A and Djurovic, S and Dale, AM and Andreassen, OA and Einevoll, GT},
+  booktitle={Society for Neuroscience Meeting},
+  year={2017},
+}
+@article{maeki2017c,
+title={A stepwise neuron model fitting procedure designed for recordings with high spatial resolution: application to layer 5 pyramidal cells. }
+author={M\"aki-Marttunen, T and Krull, F and Bettella, F and Metzner, C and Devor, A and Djurovic, S and Dale, AM and Andreassen, OA and Einevoll, GT}
+journal={\"a}
```

Comment:
Is this OK @ChristophMetzner?

Reply:
Looks like I made a mistake in the journal for the last entry (maeki2017c),  this should be 'Journal of Neuroscience Methods'

Reply:
I thought so - I'll correct it. Gonna go merge now.